collect_app/src/main/res/values/strings.xml

```diff
@@ -397,4 +397,28 @@
 <string name="geotrace_view_change_location">View or Change GeoTrace</string>
 <string name="bearing">Bearing: %.3f</string>
 <string name="direction">Direction: %s</string>
+<string name="select_odk_shortcut">Select ODK Shortcut</string>
+<string name="bad_uri">Bad URI: </string>
+<string name="unrecognized_uri">Unrecognized URI: </string>
+<string name="no_form_loaded">no form loaded</string>
+<string name="multiple_media_folders_detected_notification">More than one media folder detected, please remove one and try again</string>
+<string name="settings_successfully_loaded_file_notification">Settings successfully loaded from file</string>
+<string name="corrupt_settings_file_notification">Settings successfully loaded from file</string>
+<string name="selected_invalid_image">Selected file is not a valid image</string>
+<string name="edited_osm_file">Edited OSM XML File:</string>
+<string name="invalid_osm_data">Something went wrong. We did not get valid OSM data.</string>
+<string name="alert">Alert</string>
+<string name="install_openmapkit">Please install OpenMapKit!</string>
+<string name="go_back">Go back</string>
+<string name="download_selected">Download Selected</string>
+<string name="detection_frequency">detection frequency</string>
+<string name="two_hundred">200</string>
+<string name="text1">text1</string>
+<string name="text2">text2</string>
+<string name="record_geopoint">Record GeoPoint</string>
+<string name="gps_result" formatted="false">Latitude: %1$s\nLongitude: %2$s\nAltitude: %3$sm\nAccuracy: %4$sm</string>
```

Comment:
Do you know what the formatted="false" does in this context? I expected it to not do the substitution but evidently it does!